tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -836,41 +873,23 @@ def cleanup_storage(self):
                     self.device.file_manager.remove('/'.join([path, item]))
 
     def cleanup_gaia(self, full_reset=True):
-        # unlock
-        if self.data_layer.get_setting('lockscreen.enabled'):
-            self.device.unlock()
-
-        # kill the FTU and any open, user-killable apps
-        self.apps.kill_all()
-
         if full_reset:
-            # disable passcode
-            self.data_layer.set_setting('lockscreen.passcode-lock.code', '1111')
-            self.data_layer.set_setting('lockscreen.passcode-lock.enabled', False)
-
-            # change language back to english
-            self.data_layer.set_setting("language.current", "en-US")
-
-            # reset keyboard to default values
-            self.data_layer.set_setting("keyboard.enabled-layouts",
-                                        "{'app://keyboard.gaiamobile.org/manifest.webapp': {'en': True, 'number': True}}")
+            # kill the FTU and any open, user-killable apps
+            self.apps.kill_all()
 
-            # reset do not track
-            self.data_layer.set_setting('privacy.donottrackheader.value', '-1')
+            # unlock
+            if self.data_layer.get_setting('lockscreen.enabled'):
+                self.device.unlock()
 
-            if self.data_layer.get_setting('airplaneMode.enabled'):
-                # enable the device radio, disable airplane mode
-                self.data_layer.set_setting('airplaneMode.enabled', False)
-
-            # Re-set edge gestures pref to False
-            self.data_layer.set_setting('edgesgesture.enabled', False)
+            defaults = DEFAULT_SETTINGS.copy()
```

Comment:
Looks like this will clobber any customizations we made from modifyDefaults(). We could simply call that again here, before updating the data_layer, or could attempt to refactor it out to SPOT.